packages/dbos-cloud/login.ts

```diff
@@ -123,7 +123,7 @@ export async function login(host: string): Promise<number> {
   const bearerToken = "Bearer " + token;
   try {
     const response = await axios.get(
-      `https://${host}/v1alpha1/user`,
+      `https://${host}/v1alpha1/user`, // TODO: Change to /user/profile endpoint
```

Comment:
We can change this endpoint in this PR because we'll upgrade cluster to support this soon. Do you want to put it in a separate PR?